staging/src/k8s.io/apiserver/pkg/server/options/authentication.go

```diff
@@ -49,6 +51,35 @@ type RequestHeaderAuthenticationOptions struct {
 	AllowedNames        []string
 }
 
+func (s *RequestHeaderAuthenticationOptions) Validate() []error {
```

Comment:
I think this may need to be included in https://github.com/kubernetes/kubernetes/blob/83681764588b36301909c8fffdb7bc9083b62cf3/staging/src/k8s.io/apiserver/pkg/server/options/authentication.go#L199

Also, while it seems like quite an obvious change, some unit tests might be useful.

Reply:
Yeah I did not see where this is called.